packages/apputils-extension/src/announcements.ts

```diff
@@ -240,9 +240,9 @@ export const announcements: JupyterFrontEndPlugin<void> = {
               let notificationId: string;
               options.actions = [
                 {
-                  label: trans.__('Do not check for updates'),
+                  label: trans.__('Disable update notifications'),
```

Comment:
For the news the text used is:

```
                    label: trans.__('Hide'),
                    caption: trans.__('Never show this notification again.'),
```

Should we use a similar wording?

Reply:
I had proposed "Ignore" in https://github.com/jupyterlab/jupyterlab/issues/16608#issuecomment-2247142106, but @krassowski was concerned that a one-word label like "Hide" or "Ignore" could lead users to believe that it was a one-time dismissal, without the connotation of "never show this notification again". I personally prefer shorter button labels, though.

Reply:
What about `Ignore all` or `Ignore all updates`?

Reply:
_Ignore all updates_ could be a good candidate. _Ignore all_ sounds too generic and may raise questions about what will be ignored.

Reply:
@JasonWeill shall we settle on "Ignore all updates" as per above suggestion?